rtengine/histmatching.cc

```diff
@@ -213,6 +213,7 @@ void RawImageSource::getAutoMatchedToneCurve(std::vector<double> &outCurve)
             histMatchingCache = outCurve;
             return;
         }
+        skip = LIM(5, skip * fh / h, 10); // adjust the skip factor -- the larger the thumbnail, the less we should skip to get a good match
```

Comment:
for loadfromraw() skip % 2 should be zero, because in loadfromraw() it will be adjusted to a multiple of 2 to match the bayer pattern. https://github.com/Beep6581/RawTherapee/blob/dev/rtengine/rtthumbnail.cc#L464.

I just saw a possible bug in loadfromraw where skip can be set to 1. I will prepare a fix for that.